src/sssom/validators.py

```diff
@@ -4,8 +4,11 @@
 from typing import List
 
 from jsonschema import ValidationError
-from linkml.validators.jsonschemavalidator import JsonSchemaDataValidator
-from linkml.validators.sparqlvalidator import SparqlDataValidator  # noqa: F401
+
+# from linkml.validators.jsonschemavalidator import JsonSchemaDataValidator
+# from linkml.validators.sparqlvalidator import SparqlDataValidator  # noqa: F401
+from linkml_runtime.processing.referencevalidator import ReferenceValidator
```

Comment:
What is ReferenceValidator?

Reply:
Validates a reference (schema). Takes `SchemaView` as an input

Reply:
Does this cover all of what the JSON schema checker was doing previously? What else does it do? Do you know why it was renamed?

Reply:
`ReferenceValidator` is a generalized validator that validates whatever is passed through and automatically detects the format. Hence it substitutes JsonSchema and SPARQL.

Reply:
Ok, I read all of it and don't understand it. If you understand it well enough to know that this change actually does at least as many checks as the previous JSON-specific thing, I am fine. RIght now, I am not clear wether that reference implementation actually runs all the JSON schema checks...

Reply:
[test_validate.py](https://github.com/mapping-commons/sssom-py/blob/sssom-using-cookiecutter/tests/test_validate.py#L23-L39) confirms that it works.